Updates per CELA feedback (#2670)
Azure/azure-sdk: docs/policies/support.md

## **Azure SDK lifecycle and support policy**

Azure SDK Lifecycle and support are governed by the latest [Microsoft Modern Lifecycle Policy](https://docs.microsoft.com/en-US/lifecycle/policies/modern),  which will prevail in case of any conflicts with the information below. 

### **Package lifecycle** 
Here are the stages of a typical package lifecycle (for major versions)

> 1.	**Beta** – A new SDK is available for early access and feedback purposes and not recommended to be used in production.  

Reviewer: ```suggestion
1.	**Beta** – A new SDK that is available for early access and feedback purposes and is not recommended for use in production. 
```
